### PR TITLE
Check mastership

### PR DESCRIPTION
**Pull request by gdbelvin**

### CODE REVIEW

File: core/sequencer/server.go [hunk 1]
```diff
@@ -327,6 +327,10 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 		return nil, err
 	}
 
+	// Verify that we are still master.
+	if err := ctx.Err(); err != nil {
```
Comment (pavelkalinnikov) [inline core/sequencer/server.go]: Would `SetLeavesAtRevision` not return if `ctx` is done?

Comment (gdbelvin) [reply core/sequencer/server.go]: I hope so, I can't seem to actually find the code that would do that

Comment (pavelkalinnikov) [reply core/sequencer/server.go]: I guess it should be in grpc code that underpins the call?

**Comment by gdbelvin** [comment]
Redundant PR

**Pull request closed without merging**